src/scss/password-reset-page.scss

```diff
@@ -0,0 +1,64 @@
+@import 'variables';
+
+.password-reset-page {
+  background-image: url('../assets/Splash.png');
+  background-size: contain;
+
+  .card-container {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.5);
+    backdrop-filter: blur(4px);
+  }
+
+  .reset-card-wrapper {
+    width: 500px;
+    max-width: 500px;
+  }
+
+  .reset-card {
+    margin: 8px;
+    border-radius: 8px;
+    padding: 32px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .card-title {
+    font-weight: 600;
+    margin: 0;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    width: 80%;
+    text-align: center;
+    margin: 15px;
+  }
+
+  .card-text {
+    text-align: center;
+  }
+
+  .textbox {
+    margin: 15px;
```

Comment:
I think the inputs could use a bit less top and bottom margin, maybe something like `15px` for left and right and `8px` for top and bottom, then you could give the submit button some top margin,  just my opinion really

Reply:
I adjusted the vertical margin and just made the horizontal margin 0 since it didn't really matter